internal/format/format.go

```diff
@@ -65,10 +65,14 @@ func HumanDurationWithCase(d time.Duration, useCaps bool) string {
 }
 
 func HumanTime(t time.Time, zeroValue string) string {
-	return HumanTimeWithCase(t, zeroValue, true)
+	return humanTimeWithCase(t, zeroValue, true)
 }
 
-func HumanTimeWithCase(t time.Time, zeroValue string, useCaps bool) string {
+func HumanTimeLower(t time.Time, zeroValue string) string {
```

Comment:
Functions with boolean arguments are generally not great because it's not clear what that boolean means from the caller.

We already have an exported function for using uppercase, so this changes it to handle the only other remaining case (lowercase the first letter).